src/lib/umi/sendOptions.ts

```
@@ -7,6 +7,8 @@ export enum ConfirmationStrategy {
 
 export interface UmiSendOptions {
   priorityFee?: number | undefined
+  /** Override the default compute unit limit (400_000). */
+  computeUnitLimit?: number | undefined
```

ESLint

[error] Expected "computeUnitLimit" to come before "priorityFee"

(perfectionist/sort-interfaces)